pkg/webui/components/steps/step/step.styl

```
@@ -0,0 +1,116 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.step
+  display: flex
+  align-items: flex-start
+
+  &:not(:last-child)
+    margin-right: $cs.xs
+
+    .content > .title::after
+      display: block
+
+.active
+  .status,
+  .content > .title
+    font-weight: 600
+
+.current
+  .status
+    color: white
+    background-color: $c-active-blue
+
+  .content > .title
+    color: $tc-black
+
+  .content > .description
+    color: $tc-deep-gray
+
+.wait
+  .status
+    opacity: .7
+    background-color: white
+    border: 1px solid $tc-subtle-gray
+    color: $tc-subtle-gray
+
+  .content > .title
+    color: $tc-subtle-gray
+
+  .content > .description
+    color: $tc-subtle-gray
+
+.success
+  .status
+    background-color: $c-active-blue
+    color: white
+
+  .content > .title
+    color: $tc-deep-gray
+
+  .content > .description
+    color: $tc-subtle-gray
+
+  .content > .title::after
+    background: $c-active-blue
+
+.success.transition-failed
+  .content > .title::after
+    background: $c-error
+
+.failure
+  .status
+    background-color: $c-error
+    color: white
+
+  .content > .title,
+  .content > .description
+    color: $c-error
+
+.status
+  height: 2rem
+  width: 2rem
+  border-radius: 2rem
+  text-align: center
+  margin-right: $cs.xs
+  transition: background-color $ad.m
+
```

```suggestion
  box-sizing: border-box

```
Will get rid of the jumping when the status changes.